src/App.test.js

```diff
@@ -67,15 +67,15 @@ describe('Fazer uma requisição para o endpoint `/planets` da API de Star Wars
     };
   });
 
-  it('renderiza uma tabela com 13 colunas', async () => {
+  it('a tabela deve ter 13 colunas', async () => {
```

Comment:
```suggestion
  it('A tabela deve ter 13 colunas', async () => {
```

Vale revisar _todos_ os its e as respectivas representações no README pra garantir que todos começam bonitinho com caixa alta!